unit-tests/func/serializable-device/test-json-load.py

```diff
@@ -0,0 +1,51 @@
+import sys
+sys.path.insert(0,"C://Users//mmirbach//git//librealsense//build//RelWithDebInfo")
+sys.path.insert(0,"C://Users//mmirbach//git//librealsense//unit-tests//py")
+
+from time import sleep
+import pyrealsense2 as rs, test
+
+devices = test.find_devices_by_product_line_or_exit(rs.product_line.L500)
+device = devices[0]
+depth_sensor = device.first_depth_sensor()
+sd = rs.serializable_device(device)
+
+visual_preset_number = depth_sensor.get_option(rs.option.visual_preset)
+visual_preset_name = rs.l500_visual_preset(int(visual_preset_number))
+
+#############################################################################################
+if visual_preset_name == rs.l500_visual_preset.default:
```

Comment:
How would we get to a state where it's not the default?

Reply:
If the camera is set to a different preset by a user before the test is run

Reply:
Hmmm... and when Avishag removes the default preset?
Any way to write this so she doesn't have to go in and change it?